### PR TITLE
FloatingButton - Remove UNSAFE method

### PR DESCRIPTION
**Pull request by Inbal-Tish**
FloatingButton - Turning component to a fully controlled component by removing state management.

No breaking changes!

### CODE REVIEW

File: src/components/floatingButton/index.tsx [hunk 1]
```diff
@@ -57,44 +51,21 @@ const gradientImage = () => require('./gradient.png');
...
+  
   constructor(props: FloatingButtonProps) {
     super(props);
 
-    this.state = {
-      shouldAnimateHide: false,
-      isVisible: props.visible,
-      animating: false
-    };
+    this.initialState = props.visible;
```
Comment (M-i-k-e-l) [inline src/components/floatingButton/index.tsx:63]: Maybe `initialVisibility`?

**Review by M-i-k-e-l — COMMENTED** [review]
Hiding does not work when `withoutAnimation = true`

File: src/components/floatingButton/index.tsx (hunk 1 above)
Comment (Inbal-Tish) [reply src/components/floatingButton/index.tsx]: Done

**Comment by M-i-k-e-l** [comment]
> Hiding does not work when `withoutAnimation = true`

I think you've missed this one.

**Comment by Inbal-Tish** [comment]
@M-i-k-e-l Thanks! Good catch! Fixed.

**Review by M-i-k-e-l — APPROVED** [review]

**Pull request merged**